extras/postman/derbynet.postman_collection.json

```diff
@@ -0,0 +1,523 @@
+{
+	"info": {
+		"_postman_id": "30e6dc99-f349-488d-b707-99d9fd9d24c2",
+		"name": "derbynet",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Token",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "952abdb8-0f76-460d-9370-822c07ce00c1",
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"id": "0a25a171-ac18-4068-ba4c-9843dacd3060",
+						"exec": [
+							"//Successful login is shown with a session cookie (or if i already have one) and <success>{{username}}</success>",
+							"",
+							"//check I get a cookie if i don't already have one",
+							"if (!pm.cookies.has(\"PHPSESSID\")){",
+							"    pm.test(\"Did I get a cookie?\", function () {",
+							"        pm.response.to.have.header(\"Set-Cookie\") || pm.expect(pm.cookies.has(\"PHPSESSID\")).to.be.true;",
+							"    });",
+							"};",
+							"",
+							"pm.test(\"Check a cookie is now set \", function () {",
+							"    pm.expect(pm.cookies.has(\"PHPSESSID\")).to.be.true;",
+							"});",
+							"",
+							"//Check my user exists",
+							"pm.test(\"Recognises my username\", function () {",
+							"    pm.expect(pm.response.text()).to.include(\"<success>\"+pm.environment.get(\"username\")+\"</success>\");",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "User-Agent",
+						"value": "Java/1.8.0_25",
+						"type": "text",
+						"disabled": true
+					}
+				],
+				"body": {
+					"mode": "urlencoded",
+					"urlencoded": [
```

Comment:
Although application/x-www-form-urlencoded is the most common content-type, and probably the only one presently sent from derby-timer.jar, multipart/form-data would also be understood by either Apache or nginx.  So I'm not sure if this isn't slightly too specific.